### ISSUE TITLE
Swipe : swiper between two WebGL tiles does not work

### ISSUE THREAD

**Issue by Dolpic** [Viglino/ol-ext#1051]
Issue : 
When creating a swiper between two layers using WebGL tiles, the swiper only displays one of the two layers.

Expected behavior:
The map should print the two layers, with the swiper between them. Here, only one of the two layers is shown on the map.

Cause:
Before being rendered, each layer clears the whole canvas. The last layer clears the other one and then is drawn.
Each layer should only clear its side of the canvas.

A solution is implemented in #1052

**Comment by Viglino** [comment]
Actually it doesn't work as you have to clear the other part of the screen when moving back the slider...
We almost have to clear the whole canvas on the first layer.

https://github.com/Viglino/ol-ext/assets/7868217/e83c98e9-476c-4893-9ee0-679c41777a61

**Comment by Viglino** [comment]
I finally got it using the frameState time to clear the whole layer once a frameState 😉 
Thank-you for the boost 💡 
```javascript
if (this._time != e.frameState.time) {
  ctx.clearColor(0, 0, 0, 0);
  ctx.clear(ctx.COLOR_BUFFER_BIT);
  this._time = e.frameState.time
}
```

**Comment by Dolpic** [comment]
> I finally got it using the frameState time to clear the whole layer once a frameState 😉 Thank-you for the boost 💡
> 
> ```js
> if (this._time != e.frameState.time) {
>   ctx.clearColor(0, 0, 0, 0);
>   ctx.clear(ctx.COLOR_BUFFER_BIT);
>   this._time = e.frameState.time
> }
> ```

Nice ! Thanks !
I also created a new PR (#1054) that should lead to the same result, but doesn't require the use of the time.
For me, both solutions seem to work.

**Comment by Dolpic** [comment]
I close this issue as it was solved with #1052. Thanks for the fix !
